setup.cfg

```diff
@@ -1,11 +1,30 @@
 [flake8]
-select = E,F,W,N,D
+select = E,F,W,N,D,RST
 ignore = W503,E126,E133,D107
```

Comment:
I personally find it quite helpful to add comments on why specific errors are ignored, it saves me having to look these errors up in the relevant tables online. Moreover, while sometimes the reasons are stylistic, other times it's to allow for certain specific edge cases that eventually become irrelevant at which point it's easier to see that the ignores can be removed.